virttest/utils_disk.py

```diff
@@ -601,6 +603,165 @@ def create_filesystem(session, partition_name, fstype, ostype, timeout=360):
         create_filesyetem_linux(session, partition_name, fstype, timeout)
 
 
+def get_mpoint_fstype_linux(session, partition):
+    """
+    Get a partition's file system type and mountpoint.
+
+    :param session: session object to guest.
+    :param partition: disk partition, like /dev/sdb1.
+    :return: Tuple (mountpoint, fstype)
+    """
+    mount_list = session.cmd_output('cat /proc/mounts')
+    mount_info = re.search(r'%s\s(.+?)\s(.+?)\s' % partition, mount_list)
+    if mount_info:
+        return mount_info.groups()
+    raise exceptions.TestError('partition is not mounted, mounted'
+                               'list as:\n%s' % mount_list)
```

Comment:
Library functions should return a negative value instead of commit a `TestError` when the result is unexpected.

Reply:
thanks, updated~